apps/site/lib/site_web/templates/fare_transformation/_nearby_locations.html.eex

```diff
@@ -0,0 +1,46 @@
+<div class="c-location-cards c-location-cards--background-gray small-set grouped">
+  <%= for {%Fares.ProposedLocations.Location{stop_name: name, retail_fvm: retail_fvm, latitude: latitude, longitude: longitude} = _location, distance} <- @nearby_proposed_locations do %>
+
+    <div class="c-location-card c-location-card--resp">
+
+      <div class="c-full-width">
```

Comment:
If you remove `display: flex` from the parent then this wrapper doesn't seem to be needed.